server.js

```diff
@@ -119,7 +119,7 @@ scServer.on('warning', function (err) {
 });
 
 if (AUTH_KEY) {
-  scServer.addMiddleware(scServer.MIDDLEWARE_HANDSHAKE, (req, next) => {
+  scServer.addMiddleware(scServer.MIDDLEWARE_HANDSHAKE_SC, (req, next) => {
```

Comment:
Ah yep nice catch ;p
It should be `MIDDLEWARE_HANDSHAKE_WS` though (which happens sooner and offers better performance). You're right though that the error won't be visible on the client-side but performance is more important in this case.